src/Services/ScormService.php

```diff
@@ -252,28 +243,19 @@ private function generateScorm(UploadedFile $file): array
         $hashName = Uuid::uuid4();
         $hashFileName = $hashName . '.zip';
         $scormData = $this->parseScormArchive($file);
-        $this->unzipScormArchive($file, 'scorm/' . $scormData['version'] . '/' . $hashName);
+        $scormFilePath = 'scorm/' . $scormData['version'] . '/' . $hashName;
+        $this->unzipScormArchive($file, $scormFilePath);
 
         if (!config()->has('filesystems.disks.' . config('scorm.disk') . '.root')) {
             throw new StorageNotFoundException();
         }
 
-        $rootFolder = config('filesystems.disks.' . config('scorm.disk') . '.root');
-
-        if (substr($rootFolder, -1) != '/') {
-            // If end with xxx/
-            $rootFolder = config('filesystems.disks.' . config('scorm.disk') . '.root') . '/';
-        }
-
-        $destinationDir = 'scorm/' . $scormData['version'] . '/' . $rootFolder . $hashName; // file path
-
-        // Move Scorm archive in the files directory
-        $finalFile = $file->move($destinationDir, $hashName . '.zip');
+        Storage::putFileAs($scormFilePath, $file, $hashFileName . '.zip');
```

Comment:
As above, shouldn't this be
```suggestion
        Storage::disk('scorm.disk')->putFileAs($scormFilePath, $file, $hashFileName . '.zip');
```